### PR TITLE
Feature/5110 account details base screen

### PR DESCRIPTION
**Pull request by surajeaton**
Fixes # BLUI-5110

#### Changes proposed in this Pull Request:

- Added account details base screen, tests, docs
- Added example for account details base screen

#### Screenshots / Screen Recording (if applicable)

- 
Simulator Screenshot - iphone 12 pro - 2024-02-15 at 20 09 21

#### To Test:

- cd login-workflow
- yarn start:example
- Open Drawer 
- Press Account Details Base

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/src/screens/AccountDetailsScreen/AccountDetailsScreenBase.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        errorText: {
+            color: theme?.colors.error,
+            letterSpacing: 0,
+        },
+    });
+
+/**
+ * Component renders error text below the input field.
+ *
+ * @param errorText  The text of the error.
```
Comment (JeffGreiner-eaton) [inline login-workflow/src/screens/AccountDetailsScreen/AccountDetailsScreenBase.tsx:34]: you can update this to @param {AccountDetailsScreenProps}....

**Comment by JeffGreiner-eaton** [comment]
can you update the running example to not set initial first name and last name

**Comment by ArshdeepSingh3-eaton** [comment]
on iOS i am able to scroll the screen
even though  it should not happen

update scrollview prop - ' bounces={false} '

**Comment by ArshdeepSingh3-eaton** [comment]
everything else looks good

**Comment by surajeaton** [comment]
> can you update the running example to not set initial first name and last name and test instructions to use ErrorManager

I have added commented code for error logic. To test, please uncomment lines 10-12 and 36.

**Review by JeffGreiner-eaton — APPROVED** [review]
looks good

image

**Pull request merged**